cmake/CMakeLists.txt

```diff
@@ -53,7 +53,7 @@ include(CheckCCompilerFlag)
 include(CheckIncludeFileCXX)
 
 if(${CMAKE_CXX_COMPILER_ID} STREQUAL "Intel")
-  set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -restrict")
+  set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -restrict -std=c++11")
```

Comment:
Do all intel compiler versions support this flag?

Reply:
It is supported going back to at least 2015 compiler versions.

Reply:
We require version 16 in https://github.com/lammps/lammps/blob/master/cmake/Modules/Packages/USER-INTEL.cmake#L38-L40, so is that 2016?

Reply:
Yes.